gammapy/utils/regions.py

```diff
@@ -197,7 +197,7 @@ def make_orthogonal_rectangle_sky_regions(start_pos, end_pos, wcs, height, nbin=
     return regions
 
 
-def make_concentric_annulus_sky_regions(center, radius_max, nbin=11):
+def make_concentric_annulus_sky_regions(center, radius_max, radius_min=1e-5 * u.deg, nbin=11):
```

Comment:
another solution would be to make the first region a disk

Reply:
Yes, however this might become tricky later for bookkeeping: e.g. in #3863 we would then need to handle different shapes of regions for flux profiles in addition. This might impose additional requirement on serialization of flux profiles etc. So I prefer to keep the consistent region shape for, such that the parameters of the individual regions are consistent.